core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.ts

```diff
@@ -1,12 +1,19 @@
 import { AfterViewInit, Component, Input, OnDestroy } from '@angular/core';
 import * as c3 from 'c3';
-import { PrimitiveArray } from 'c3';
+import {Primitive, PrimitiveArray} from 'c3';
```

Comment:
the frontend `ts` files' format is off. we can resolve it offline. Intellij and vscode have some differences in their default formatting.